README.md

```diff
@@ -1,48 +1,162 @@
 # feedback-DT
 
-## Creating a BabyAI dataset
-Currently, this can be run with three different parameters. The plan is to also add an RGB parameter.
-```src/_datsets.py --env_name --num_episodes --include_timeout```
+This repository houses all the code for our paper '*Is Feedback All You Need? Leveraging Natural Language Feedback in Goal-Conditioned Reinforcement Learning*', as well as ongoing work developing the project further. If you use any of the code for your sown work, please cite the paper as
```

Comment:
sown -> own
Should we mention that the NeurIPS 2023 GCRL submission code can be found in the neurips_gcrl branch?

Reply:
good idea